### PR TITLE
[U1] Frontend: Login - Navigation

### PR DESCRIPTION
**Pull request by germangelv**
## Summary
- Added navigations in signup and hyperlink in terms 

## Trello Card
https://trello.com/c/d5MMUFtV/

## Screenshot
ezgif-2-82d1956b43e2

### CODE REVIEW

File: app/src/main/java/ar/com/wolox/android/example/ui/login/LoginPresenter.java [hunk 1]
```diff
@@ -17,15 +17,15 @@
     LoginPresenter() {
     }
 
-    public void setPreferences(Context context, String emailField) {
+    void setPreferences(Context context, String emailField) {
```
Comment (igsosa92) [inline app/src/main/java/ar/com/wolox/android/example/ui/login/LoginPresenter.java]: I think that shared preferences can be inject, so you dont need to pass context to presenter to call shared preferences. We can talk about it together.

File: app/src/main/java/ar/com/wolox/android/example/ui/login/LoginPresenter.java [hunk 2]
```diff
@@ -17,15 +17,15 @@
...
         sharedPref = context.getSharedPreferences(context.getString(R.string.vlogin_preferences_name), Context.MODE_PRIVATE);
         editor = sharedPref.edit();
         editor.putString("username", emailField);
```
Comment (igsosa92) [inline app/src/main/java/ar/com/wolox/android/example/ui/login/LoginPresenter.java]: "username" can be set on a constant value. This is from the last PR

File: app/src/main/res/layout/fragment_signup.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:gravity="center"
+        android:text="@string/vsignup__activity"
```
Comment (igsosa92) [inline app/src/main/res/layout/fragment_signup.xml]: remember to remove the first 'v' and in this case, the doble '__'

File: app/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -33,6 +37,7 @@
             android:name=".example.ui.viewpager.ViewPagerActivity"
             android:screenOrientation="portrait" />
 
+
```
Comment (igsosa92) [inline app/src/main/AndroidManifest.xml]: delete this blank line

File: app/src/main/java/ar/com/wolox/android/example/ui/home/HomeFragment.java [hunk 5]
```diff
@@ -1,18 +1,28 @@
 package ar.com.wolox.android.example.ui.home;
 
+import android.content.Intent;
+
```
Comment (igsosa92) [inline app/src/main/java/ar/com/wolox/android/example/ui/home/HomeFragment.java]: delete this blank line

File: app/src/main/java/ar/com/wolox/android/example/ui/home/HomePresenter.java [hunk 6]
```diff
@@ -1,11 +1,13 @@
 package ar.com.wolox.android.example.ui.home;
 
 import javax.inject.Inject;
+
```
Comment (igsosa92) [inline app/src/main/java/ar/com/wolox/android/example/ui/home/HomePresenter.java]: delete this blank line

File: app/src/main/java/ar/com/wolox/android/example/ui/home/HomePresenter.java [hunk 7]
```diff
@@ -1,11 +1,13 @@
...
 import ar.com.wolox.wolmo.core.presenter.BasePresenter;
 
 public class HomePresenter extends BasePresenter<HomeView> {
 
     @Inject
     HomePresenter() {
     }
+
```
Comment (igsosa92) [inline app/src/main/java/ar/com/wolox/android/example/ui/home/HomePresenter.java]: delete this blank line

File: app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java [hunk 8]
```diff
@@ -1,13 +1,17 @@
 package ar.com.wolox.android.example.ui.login;
 
 import android.content.Intent;
+import android.text.method.LinkMovementMethod;
 import android.view.View;
 import android.widget.Button;
 import android.widget.EditText;
-import android.widget.Toast;
+import android.widget.TextView;
+
```
Comment (igsosa92) [inline app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java]: delete this blank line

File: app/src/main/res/values/strings.xml [hunk 9]
```diff
@@ -25,5 +25,7 @@
     <string name="login_terms_and_conditions">By joing you agree to our <u>Terms &amp; Conditions</u></string>
     <string name="home_activity">HOME</string>
     <string name="password_required">Password is Required</string>
+    <string name="root_username">username</string>
```
Comment (igsosa92) [inline app/src/main/res/values/strings.xml]: Where do you use this now? I think you can delete it

File: app/src/main/java/ar/com/wolox/android/example/ui/home/HomePresenter.java (hunk 6 above)
Comment (germangelv) [reply app/src/main/java/ar/com/wolox/android/example/ui/home/HomePresenter.java]: This line is generated by Android Studio when I reformat code

**Pull request merged**